src/main/java/io/OutputFileWriter.java

```diff
@@ -0,0 +1,39 @@
+package io;
+
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class OutputFileWriter {
+
+    public void writeOutputFile() {
+        Workbook workbook = new XSSFWorkbook();
+        Sheet sheet = workbook.createSheet("Output");
+
+
+        int rowNum = 0;
+
+        for (String value : InputFileReader.data) {
```

Comment:
`InputFileReader.data` cant say its the most elegant solution. Why address class field if you can call its method?

Reply:
Fixed